### PR TITLE
Make base_dir input params

### PR DESCRIPTION
**Pull request by ajlee21**
This PR adds "base_dir" as a params in the config file.

Addresses: https://github.com/greenelab/ponyo/issues/43

Main changes:
1.  Updated the README file
2.  Updated ponyo/ scripts

Remaining files were rerun but without any changes

### CODE REVIEW

**Comment by ajlee21** [comment]
Note: I tested that the notebooks run as expected and ran the pytests locally. Github actions is taking a while for some reason.

File: README.md [hunk 1]
```diff
@@ -63,13 +65,12 @@ The tables lists the core parameters required to generate simulated data using m
...
+| num_simulated_experiments*| int: The number of experiments to simulate. This parameter is needed if using the latent transformation approach - simulate_by_latent_transform.|
+| num_simulated*| int: The number of experiments to simulate (i.e. the number of times to shift the starting template experiment). This parameter is needed if using the latent transformation approach - shift_template_experiment.|
+| metadata_experiment_colname* | str: Column header that contains experiment id that maps expression data and metadata. This parameter needed if using either latent transformation approaches.|
+| metadata_sample_colname* | str: Column header that contains sample id that maps expression data and metadata. This parameter needed if using either latent transformation approaches.|
+| project_id*| int: The experiment id to use as template experiment. The id is using the values found in the <metadata_experiment_colname> column of the <metadata_filename>. This <project_id> corresponds to a group of samples that were used to test an single hypothesis. This parameter is needed if using either latent transformation approaches.|
```
Comment (taylorreiter) [inline README.md:73]: I think the second sentence here might not make sense? at least the way it is written is hard for me to understand what it means

**Review by taylorreiter — APPROVED** [review]
I think this looks great! I have one small comment about the README, the rest of the clarifications look wonderful. Thanks Alex!

**Comment by ajlee21** [comment]
The current test notebooks timed out, in the next PR I will remake the test notebooks to be faster.

**Pull request merged**